AboutPage.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+<head>
+	<title>About Pet Adoption></title> 
+	<meta charset="utf8">
+</head>
+<body>
```

Comment:
add this here
<!--<!-- Create Nav Bar -->
        <div class="nav">
            <a class="active" href="main.html">Home</a>
            <a href="pets.html">Pets</a>
            <a href="adoption.html">Adoption</a>
            <a href="about.html">About</a>
        </div>
        -->